src/modules/lecturesight-framesource-impl/src/main/resources/OSGI-INF/cv.lecturesight.framesource.impl.FrameSourceManagerImpl.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0"
+               name="lecturesight.framesource.manager"
+               immediate="true">
+  <implementation class="cv.lecturesight.framesource.impl.FrameSourceManagerImpl"/>
+  <property name="service.description" value="FrameSourceManager Implementation"/>
+  <service>
+    <provide interface="cv.lecturesight.framesource.FrameSourceManager"/>
+    <provide interface="org.osgi.service.event.EventHandler"/>
+  </service>
+
+  <property name="event.topics">
+      org/osgi/framework/ServiceEvent/REGISTERED
+      org/osgi/framework/ServiceEvent/UNREGISTERING
+  </property>
+  <reference name="config"
```

Comment:
This file could use the more compact format used elsewhere for consistency.

Reply:
Thanks, fixed.